### PR TITLE
feat(presentational): add paragraph component

### PR DESCRIPTION
**Pull request by agory**

### CODE REVIEW

File: src/web/components/Paragraph/Paragraph.test.js [hunk 1]
```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+import { shallow } from 'enzyme';
+import Paragraph from './';
```
Comment (mfrachet) [inline src/web/components/Paragraph/Paragraph.test.js]: Well, can we use `Text` this time ? 😊

Comment (mfrachet) [reply src/web/components/Paragraph/Paragraph.test.js]: The file name is Paragraph :p

Comment (agory) [reply src/web/components/Paragraph/Paragraph.test.js]: Oh :hankey: . the same mistake sorry

**Comment by mfrachet** [comment]
I can't see my last coment, but the filenames are bad :)

**Pull request merged**